src/common/interop/HotkeyManager.cpp

```diff
@@ -56,7 +56,7 @@ bool HotkeyManager::FilterKeyboardProc(KeyboardEvent ^ ev)
     pressedKeys->Ctrl = GetAsyncKeyState(VK_CONTROL) & 0x8000;
     pressedKeys->Alt = GetAsyncKeyState(VK_MENU) & 0x8000;
     pressedKeys->Shift = GetAsyncKeyState(VK_SHIFT) & 0x8000;
-    pressedKeys->Key = ev->key;
+    pressedKeys->Key = static_cast<unsigned char>(ev->key);
```

Comment:
Is this losing data possibly?

Reply:
As far as I can tell, it is not, the keys are defined in winuser.h and go up to 0xFF.
So unsigned char would be ok.